.nais/config-prod.json

```diff
@@ -5,6 +5,8 @@
 	],
 	"service-user-path": "/serviceuser/data/prod/srvsoknadsfillager",
 	"database-name": "db-soknadsfillager",
+	"disk-size": "20",
```

Comment:
Tror denne blir ignorert. Fra [docs](https://docs.nais.io/nais-application/application/#gcpsqlinstancesdisksize):
`Disk size can be changed using this field only when diskAutoresize is set to false.`